apps/artwork_purchase/helpers.coffee

```diff
@@ -0,0 +1,49 @@
+_ = require 'underscore'
+
+module.exports =
+  isEligible: (artwork) ->
```

Comment:
> Kind of gross to share a helper across 2 apps (you'll see the artwork_purchase helper to determine if a work is eligible is being shared with the artwork app). 

This is kinda an awkward situation in this brave new world. Usually, you would add logic like this to the artwork model, but since we're generally not leveraging Backbone in these cases I would say it'd be better to either move this to a component (even if that component doesn't have UI bits) to make the cross-app dependency more explicit, or better yet move this to a new field in Metaphysics.